pkgs/servers/calibre-web/default.nix

```diff
@@ -44,6 +44,7 @@ python.pkgs.buildPythonApplication rec {
     flask-wtf
     flask-limiter
     iso-639
+    jsonschema
```

Comment:
Just wondering, what required this to be added to the package? Was it just missing, or is it required for kepubify support?